test/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
```

Comment:
Nit: you can remove `WIP` from the PR name, to indicate you're ready for a review